portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
+
+public final class Comments {
```

Comment:
Please use singular "Comment" instead of Comments. If there issues using "Comment" in case it has been used else where, use CommentEntry